api/notd/block_processor.py

```diff
@@ -43,6 +43,14 @@ def __init__(self, ethClient: EthClientInterface):
         # self.contractFilter = self.ierc721Contract.events.Transfer.createFilter(fromBlock=6517190, toBlock=6517190, topics=[None, None, None, None])
         self.erc721TansferEventSignatureHash = Web3.keccak(text='Transfer(address,address,uint256)').hex()
 
+        with open('./contracts/IERC1155MetadataURI.json') as contractJsonFile:
+            contractJson = json.load(contractJsonFile)
+        self.ierc1155Contract = self.w3.eth.contract(abi=contractJson['abi'])
+        self.ierc721TransferEvent = self.ierc1155Contract.events.TransferSingle()
```

Comment:
we're gonna need to do both transfer single and transfer batch. could you paste her ewhat ths signatures are for both so we have a reference